docs/en/conditions.md

```diff
@@ -0,0 +1,68 @@
+# Conditions
+
+To control the flow of rendering use `if` `elseif` `else` tags.
+ 
+```html
+<if foo>
+  {foo}
+</if>
+```
+
+```html
+<if not foo>
+  {foo}
+</if>
+```
+
+```html
+<if foo and bar>
+  {foo}
+</if>
+```
+
+```html
+<if foo.length>
+  {foo}
+</if>
+```
+
+```html
+<if isValid()>
+  {foo}
+</if>
+```
+
+Sometimes you will need more complex test for your conditions.
+For this purposes pure-engine brings actions.
+Actions are sets of predefined attributes which put together created a test function.
```

Comment:
`Actions can be seen as a predefined conditions which follow the natural language principles.`